tasks/accounts.ts

```diff
@@ -1,5 +1,5 @@
 import { Signer } from "@ethersproject/abstract-signer";
-import { task } from "hardhat/config";
+require("@nomicfoundation/hardhat-toolbox");
```

Comment:
We can actually use `import "@nomicfoundation/hardhat-toolbox";` and it would be more consistent

Reply:
done